verilog/tools/kythe/verilog_kythe_extractor.cc

```diff
@@ -60,8 +61,7 @@ static int ExtractOneFile(absl::string_view content,
     std::cout << std::endl
               << (!parse_ok ? " (incomplete due to syntax errors): " : "")
               << std::endl;
-
-    std::cout << verilog::kythe::KytheFactsPrinter(facts_tree) << std::endl;
+    multi_file_kythe_facts_extractor.ExtractKytheFacts(facts_tree);
```

Comment:
Here you no longer print the tree. Is that intended?

Reply:
Yes we now have two different entries `KytheFactsPrinter` for processing single files which won't consider any dependencies and `MultiFileFactsExtractor` which is for multi-files and cross-referencing in files.